src/webgpu/api/validation/createRenderPipeline.spec.ts

```diff
@@ -248,6 +249,56 @@ g.test('color_formats_must_be_renderable')
     t.doCreateRenderPipelineTest(isAsync, info.renderable && info.color, descriptor);
   });
 
+g.test('formats_for_depth_stencil_state_must_be_depth_stencil_formats')
```

Comment:
nit: All new tests should have `.desc()`; please put this detail in the description, then you can use a slightly shorter test name like `depth_stencil_attachment,format`

Reply:
Done.